src/Castle.Windsor.Extensions.MsDependencyInjection/NetCoreRootScope.cs

```diff
@@ -0,0 +1,32 @@
+// Copyright 2004-2020 Castle Project - http://www.castleproject.org/
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+namespace Castle.Windsor.Extensions.MsDependencyInjection
+{
+    internal class NetCoreRootScope : NetCoreScope
```

Comment:
Having separate class makes it more distinct as it has special meaning (it's automatically created, holds singletons, etc.). It can be done different ways. None is better than the other...just different

Reply:
Great, agreed it can be done different ways.